packages/protocol/scripts/foundry/make-release-foundry.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Checks that the contract version numbers in a provided branch are as expected given
+# a released branch.
+#
+# Flags:
+# -n: The network to deploy to.
+# -b: Branch to build contracts from.
+# -p: Path that the governance proposal should be written to.
+# -i: Path to the data needed to initialize contracts.
+# -r: Path to the contract compatibility report.
+# -d: Whether to dry-run this deploy
+# -f: Private key to sign transactions from.
+# -l: Path to the canonical library mapping.
+
+NETWORK=""
+PROPOSAL=""
+BRANCH=""
+INITIALIZE_DATA=""
+REPORT=""
+FROM=""
+LIBRARIES=""
+
+while getopts 'b:n:p:i:r:f:l:F' flag; do
```

Comment:
Flag `F` is not existing
```suggestion
while getopts 'b:n:p:i:r:f:l' flag; do
```